### PR TITLE
library: Ported Popover in RUST

### PR DESCRIPTION
**Pull request by onkarrai06**
Tried to port Popover, however I am facing a problem as I have been unable to find a function to fetch the name of the popover. It says the .name() function exists, however I can't get it to work. Need some assistance.

Thanks a lot.

### CODE REVIEW

File: src/Library/demos/Popovers/code.rs [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+pub fn main() {
+    let popover_ids = ["plain_popover", "popover_menu"];
+
+    for id in popover_ids {
+        let popover: gtk::Popover = workbench::builder().object(id).unwrap();
+
+        popover.connect_closed(move |popover| {
+            let popover_name = popover.name();
+            println!("{} closed.", popover_name);
```
Comment (Hofer-Julian) [inline src/Library/demos/Popovers/code.rs:12]:
```suggestion
            let popover_name = popover.widget_name();
            println!("{popover_name} closed.");
```

**Review by Hofer-Julian — COMMENTED** [review]
This does the trick

**Comment by onkarrai06** [comment]
Changed, please check again.
@Hofer-Julian

File: src/Library/demos/Popovers/code.rs (hunk 1 above)
Comment (onkarrai06) [reply src/Library/demos/Popovers/code.rs]: Thank you.

**Review by Hofer-Julian — APPROVED** [review]

**Pull request merged**